src/databricks/labs/ucx/source_code/notebooks/migrator.py

```diff
@@ -53,7 +48,7 @@ def _apply(self, notebook: Notebook) -> bool:
                 cell.migrated_code = migrated_code
                 changed = True
         if changed:
-            self._ws.workspace.upload(notebook.path.as_posix() + ".bak", notebook.original_code.encode("utf-8"))
-            self._ws.workspace.upload(notebook.path.as_posix(), notebook.to_migrated_code().encode("utf-8"))
             # TODO https://github.com/databrickslabs/ucx/issues/1327 store 'migrated' status
+            notebook.path.rename(notebook.path.with_suffix(".bak"))
```

Comment:
shouldn't this read `notebook = notebook.replace(path=...)`?

Reply:
done